docs/csaf-vex.md

```diff
@@ -0,0 +1,497 @@
+    "namespace": "https://access.redhat.com/security/updates/classification/",
+    "text": "moderate"
+},
+```
+
+VEX metadata: 
+
+```
+"category": "csaf_vex",
+"csaf_version": "2.0",
+"distribution": {
+    "text": "Copyright © Red Hat, Inc. All rights reserved.",
+    "tlp": {
+        "label": "WHITE",
+        "url": "https://www.first.org/tlp/"
+    }
+},
+"lang": "en",
+"notes": [
+    {
+        "category": "legal_disclaimer",
+        "text": "This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original.",
+        "title": "Terms of Use"
+    }
+],
+```
+
+Vendor information:
+```
+"publisher": {
+    "category": "vendor",
+    "contact_details": "https://access.redhat.com/security/team/contact/",
+    "issuing_authority": "Red Hat Product Security is responsible for vulnerability handling across all Red Hat offerings.",
+    "name": "Red Hat Product Security",
+    "namespace": "https://www.redhat.com"
+},
+```
+
+CVE ID, CVE publish date and CVE revision history:
+
+```
+"id": "CVE-2023-20593",
+"initial_release_date": "2023-07-25T06:30:00+00:00",
+"revision_history": [
+  {
+    "date": "2023-07-25T06:30:00+00:00",
+    "number": "1",
+    "summary": "Initial version"
+  },
+  {
+    "date": "2024-04-18T04:20:20+00:00",
+    "number": "2",
+    "summary": "Current version"
+  },
+  {
+    "date": "2024-09-13T20:58:30+00:00",
+    "number": "3",
+    "summary": "Last generated version"
+  }
+],
+"status": "final",
+"version": "3"
+```
+
+### Product Tree 
+The `product_tree` section identifies all affected Red Hat software, represents the nested 
+relationship of component to product and provides CPEs or PURLs depending on the affected layer. There are two main 
+objects in the “product_tree” object: `branches` and `relationships`.
+
+#### Branches
+The parent `branches` object has one child object of the `vendor` category with the name set to "Red Hat". All 
+affected Red Hat products and components will be nested in that "branches" array. Compressed down, the parent branches 
+object would look like:
+
+```
+{
+  "branches": [
+    {
+      "branches": []
+      "category": "vendor"
+      "name": "Red Hat:
+    }
+  ]
+}
+```
+
+All nested objects included in the `branches` object of the `vendor` category fall into the following subcategories:
+* `product_family`: Represents a general Red Hat product stream and includes one or more 
+nested objects of the `product_name` category .
+* `product_name`: Represents a specific product release and is always nested under the 
+corresponding "product_family"
+* `product_version`: Represents a specific component. When displayed unnested, the 
+component is not fixed yet and will not include a specific version number. Note: This will only be present in VEX files
```

Comment:
You can also mention that components with different than FIX status do not contain purl with version information.

Reply:
This is elaborated on in the "Unfixed Product Versions (VEX only) Examples"